scripts/test

```diff
@@ -30,11 +30,11 @@ then
             app-server "./sbt scapegoat"
 
         # Run scala tests for app-server
-        docker-compose \
-            -f "${DIR}/../docker-compose.yml" run \
-            --rm \
-            --entrypoint "/bin/bash -c" \
-            app-server "./sbt test"
+#        docker-compose \
```

Comment:
It may be useful to add a reference here to the outstanding issue that causes this block to be commented out: https://github.com/azavea/raster-foundry/issues/409